data/remnant missions.txt

```diff
@@ -1252,3 +1252,237 @@ mission "Remnant: Expanded Horizons Quarg 3"
 			`	"Our historical records just have a few pictures and videos of Quarg ships, along with a basic scan that doesn't provide more than the general outline of the ship. These scans," he says while tapping the databanks, "actually picked up detailed structural layouts and even identified a number of key components. It isn't nearly enough information to be able to duplicate them, but it is enough that we can start trying to figure out how they work.`
 			`	Upon arriving at <planet> they quickly thank you, hand you <payment>, and run back to their offices to finish prepping their material to publish and distribute.`
 
+
+
+mission "Remnant: Broken Jump Drive 1"
+	name "Broken Jump Drive Delivery"
+	description "You have been asked to deliver a broken jump drive to the researchers on <planet>."
+	minor
+	landing
+	to offer
+		has "remnant blood test pure"
+	source
+		government "Remnant"
```

Comment:
Possibly you need an `attributes outfitter` as that is in your offer text

Reply:
I have omitted limiting it based on the presence of an outfitter or not, because the mission doesn't involve installing anything - just unloading (and there's rather good odds that the JDB isn't installed anyway, just stored in cargo). And unloading (or even uninstalling) is something that the player can do to other ships even floating in space without access to spaceport facilities or cargo handlers. 

I contemplated adding a "Laboratory" attribute to the current Remnant planets to do roughly the same thing without being dependent on outfitters, but again, the Remnant were doing R&D in remote archeological sites back in their early days when they were still getting established, so it is clear they can do research pretty much anywhere. One of the missions even describes how they were put in a lab on the far side of the planet in a remote location away from cities or anything to prevent collateral damage.

In short, the Remnant are like classic Sci-fi ships/groups where every ship and outpost has a science officer on the lookout for new stuff, and at least a small (but well equipped) lab tucked tucked into whatever space they have. And anyplace of any significant size has a decent size team or three.

As far as that goes, I have no plans to add any more "already existing" Remnant planets/stations anywhere, and all the "new" planets/stations that I have tentatively planned will explicitly have R&D mentioned somewhere in either the descriptions or conversations that lead to the place becoming available.

Reply:
My point is that you are possibly uninstalling the player's only means of transportation. Such a player is then stuck. And no, you cannot modify the outfits of a ship that was not capable of landing, nor can you modify the installed outfits of a ship if there is no outfitter present on the planet.

Reply:
My point about "uninstalling" was referring to the player taking outfits off another ship during boarding, not installing/uninstalling ships in their own fleet that aren't on-planet.

That being said, I'll add the outfitter requirement to the first mission along with a warning.

Reply:
OK, I've changed that. The initial mission is limited in both "to offer" and "destination" to being planets with outfitters, so the player won't get stranded, and includes a warning about not trusting them. They are *Broken* drives, after all. Highly unreliable, prone to disappearing at inconvenient times, and outfitters may not always be on hand.